plugins/modules/ldap_inc.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Philippe Duveau <pduvax@gmail.com>
+# Copyright (c) 2019, Maciej Delmanowski <drybjed@gmail.com> (ldap_attrs.py)
+# Copyright (c) 2017, Alexander Korinek <noles@a3k.net> (ldap_attrs.py)
+# Copyright (c) 2016, Peter Sagerson <psagers@ignorare.net> (ldap_attrs.py)
+# Copyright (c) 2016, Jiri Tyr <jiri.tyr@gmail.com> (ldap_attrs.py)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# The code of this module is derived from that of ldap_attrs.py
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: ldap_inc
+short_description: Use the Modify-Increment ldap V3 feature to increment an attribute value
+description:
+  - Atomically increments the value of an attribute and return its new value.
+notes:
+  - This only deals with integer attribute of an existing entry. To modify attributes
+    of an entry, see M(community.general.ldap_attrs) or to add or remove whole entries,
+    see M(community.general.ldap_entry).
+  - The default authentication settings will attempt to use a SASL EXTERNAL
+    bind over a UNIX domain socket. If you need to use a simple bind to access
+    your server, pass the credentials in O(bind_dn) and O(bind_pw).
```

Comment:
This sounds like generic information for all LDAP modules. In that case it should be part of the docs fragment?

Reply:
I agree that this paragraph is common to all ldap_xxx modules but I am not the maintainer of the other ldap_xxx modules. I do not have enough time to handle this shortly.